tests/integration/targets/openssh_keypair/tasks/setup_cryptography.yml

```diff
@@ -0,0 +1,33 @@
+---
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+- name: Attempt to install dependencies for OpenSSH > 7.8
+  block:
+    - name: Ensure cryptography >= 3.0 available
+      become: true
+      pip:
+        name: cryptography>=3.0
+        extra_args: "-c {{ remote_constraints }}"
```

Comment:
Why try to install `cryptography` when you added `setup_openssl` (which does exactly that) to `meta/main.yml`?

Reply:
Uh, I squashed all the ugly attempts at having CI clear, but the `setup_openssl` dependency, with the exception of Mac OS X, is only installing the system packaged cryptography version which is generally < 3.0 and in some cases < 2.6. I really only wanted the cryptography < 3.0 conditionals to take effect if absolutely necessary. Even with this hack the `pip` version on the following containers will not install a sufficient cryptography version: (I can check the logs again to collect which containers this task is actually `changed` for)

- CentOS 6
- CentOS 8
- openSUSE 15 py3
- Ubuntu 18.04
- RHEL 8.3
- FreeBSD 12.1
- FreeBSD 12.2
- FreeBSD 13.0

Reply:
It is intentional that the other tests run with the OS-supplied `cryptography`, otherwise we would not need all these platform tests and could stick to the `default` container :) In fact, trying to install `cryptography` with a newer version here makes this very problematic for CI of the whole collection, since other tests running later might suddenly use another `cryptography` version than the one they used to.